.prettierignore

```diff
@@ -1,2 +1,4 @@
-dist
+.vscode
```

Comment:
Prettier doesn't have an option for a global config, so I guess this has to be here.